frontend/src/ErrorPage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Link, useRouteError } from 'react-router-dom'
+
+interface ErrorType {
+  status?: number
+  statusText?: string
+  message?: string
+}
+
+const ErrorPage: React.FC = () => {
+  const error = useRouteError() as ErrorType
+
+  return (
+    <div id="error-page">
+      <h1 className="font-medium italic">
+        {error.status && error.status == 404
+          ? 'Error 404: The page you are looking for does not exist.'
+          : `Error ${error.status}: ${error.statusText}` ||
+            error.message ||
+            'Something went wrong.'}
+        <Link to="/">Go Home</Link>
```

Comment:
It's really hard to tell that this is a link since it's on the same line and has the same styling as the header. I recommend putting it on a separate line and/or changing the styling of it to look more like a link.

Reply:
Resolved in https://github.com/KirillTregubov/csc302/pull/48/commits/818f63d42ef1e887c5ce52dbec4bc0943878f66, thanks for being so thorough with your testing!